README.md

```diff
@@ -150,14 +150,17 @@ attributes to stdout. It's mainly used for testing and debugging.
 
 The propagator controls which distributed tracing header format is used.
 
-If set to a comma-separated list of the values, the multi-propagator is used. The multi-propagator attempts
-to extract the context from incoming requests using each of the configured propagator formats (in order),
-stopping after the first successful context extraction. The multi-propagator injects the context into
-outgoing requests using all the configured propagator formats.
+If set to a comma-separated list of the values, the multi-propagator is used. Special case is `baggage` 
+and `jaeger` propagators: they cannot be added to multi-propagator because they may not get a chance 
+to extract any existing Baggage, instead they can be added only to DefaultContextPropagators. 
+The multi-propagator attempts to extract the context from incoming requests using each of the 
+configured propagator formats (in order), stopping after the first successful context extraction. 
+The multi-propagator injects the context into outgoing requests using all the configured propagator 
+formats.
```

Comment:
I think this is going to be really confusing for users, suggest we just leave it out, and if anyone asks, refer them to the spec. 

```suggestion
```